src/pages/Home/index.js

```diff
@@ -336,18 +413,32 @@ const Home = () => {
         break;
       case 'version':
         let pack_format = DEFAULT_PACK_META.pack_format;
+        newPackMeta.extension = DEFAULT_EXTENSION;
+        let showResolution = false;
         switch (event.value) {
           case '1_14':
             pack_format = 4;
+            newPackMeta.fileBuilder = javaFileBuilder;
             break;
           case '1_15':
             pack_format = 5;
+            newPackMeta.fileBuilder = javaFileBuilder;
+            break;
+          case 'BR_1_14':
+            // valid, but pack_format is irrelevant
+            newPackMeta.fileBuilder = bedrockFileBuilder;
+            newPackMeta.extension = 'mcpack';
+            showResolution = true;
             break;
           default:
             console.error('Invalid pack version');
         }
         newPackMeta.pack_format = pack_format;
+        setShowResolutionSelect(showResolution);
         break;
+      case 'resolution':
+        newPackMeta.resolution = event.value;
+        break
```

Comment:
I thought prettier ran post-commit... evidently it hasn't quite worked. This line needs a semicolon.

Reply:
I have a guess that the Prettier config is not working right due to how I checked out in a restricted container (see other comment) - I'll see if I can figure that out.